### ISSUE TITLE
Cannot read properties of undefined (reading 'imageRef')     at setOverlayStyles (view.js:191:31)

### ISSUE THREAD

**Issue by donbowman** [WordPress/gutenberg#64317]
### Description

In this commit, in v18.9.0, on line 159, It adds:
```
		callbacks: {
			setOverlayStyles() {
				if ( ! state.currentImage.imageRef ) {
					return;
				}
```
https://github.com/WordPress/gutenberg/commit/71134165868#diff-540be71e7203e1f3a7c11c0cd06085f90b3b4a07f0e3bfa9efb17952d227ec96R159

state can be missing 'currentImage' in some circumstances, leading to 
```
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'imageRef')
    at setOverlayStyles (view.js:191:31)
```

### Step-by-step reproduction instructions

v6.6.1, install gutenberg v18.9.0 plugin. 
Create a page with some images (I used svg, i'm not sure if it matters)
Then resize the browser.

### Environment info

- wordpress 6.6.1
- twenty twenty four
- gutenberg v18.9.0 plugin installed.

### Please confirm that you have searched existing issues in the repo.

- [X] Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

- [X] Yes

**Comment by donbowman** [comment]
If it helps, I have uploaded the state object at the time the error happened, in json format.
[l.json](https://github.com/user-attachments/files/16517218/l.json)

**Comment by t-hamano** [comment]
Thanks for the report.

This issue has been fixed in #63995 and will ship as part of Gutenberg version 19.0.
